src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/MS/Impl/XmlNsInfo.cs

```diff
@@ -191,6 +199,16 @@ void LoadNsDefHelper(IList<XmlNsDefinition> result, string xmlns, string clrns,
             result.Add(new XmlNsDefinition { ClrNamespace = clrns, XmlNamespace = xmlns });
         }
 
+        void LoadNsDefHelper(IList<XmlNsDefinition> result, string xmlns, string clrns, string assemblyName)
+        {
+            if (String.IsNullOrEmpty(xmlns) || clrns == null)
+            {
+                throw new XamlSchemaException(SR.Get(SRID.BadXmlnsDefinition, assemblyName));
```

Comment:
Hey @dotMorten, since you pulled main, your PR build is now breaking here which is caused by my changes as part of dotnet/wpf#1. SRID is being replaced with SR which contains the same properties but they now return the resource instead of the resource name. For now, only a couple of projects uses SR instead of SRID, here's the list: https://github.com/dotnet/wpf/blob/main/eng/WpfArcadeSdk/tools/SystemResources.props#L15-L19.

```suggestion
                throw new XamlSchemaException(SR.Format(SR.BadXmlnsDefinition, assemblyName));
```

Reply:
Nice catch! Thanks!